editor/editor_node.cpp

```diff
@@ -1824,6 +1824,9 @@ void EditorNode::restart_editor() {
 		args.push_back("--disable-crash-handler");
 	}
 
+	args.push_back("--rendering-driver");
+	args.push_back(OS::get_singleton()->get_current_rendering_driver_name());
```

Comment:
I think this line needs to be changed to: `args.push_back(GLOBAL_GET("rendering/driver/driver_name"));` Otherwise, the editor restarts with the "old" driver when I change the driver in the top right drop-down menu

Reply:
Absolutely. Fixed.